Doc/install/INSTALL.rst

```diff
@@ -159,9 +159,9 @@ This will install the biopython package in your system. For checking the install
 Installing Biopython using Anaconda
 ===================================
 
-For Anaconda users, the Biopython source files are available in the `bioconda <https://bioconda.github.io/>`_ channel. To install Biopython, just do::
+For Anaconda users, the Biopython source files are available in the `conda-forge <https://conda-forge.org/>`_ channel. To install Biopython, just do::
 
-  $ conda install -c bioconda biopython
+  $ conda install -c conda-forge biopython
```

Comment:
I checked the command ``conda install biopython`` in Windows, It still fetches the 1.68 binaries uploaded to default channel. 

``conda install -c conda-forge biopython`` works really well as it fetches the latest binaries for Windows which is uploaded to the ``conda-forge`` channel.